### PR TITLE
Updates to README

### PR DESCRIPTION
**Pull request by mbbroberg**
Redesigning the README based on feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,75 @@
...
+This year we are focused on working together, whether you are contributing to an Open Source project or scripts at work. Working collaboratively together requires communication, coordination, and of course some practice with forking, branching, merging, and pull requests!
 
-## The Challenge
+## vBrownBag Schedule
 
-This year we are focused on working together, whether you are contributing to an Open Source project or scripts at work. Working collaboratively together requires communication, coordination, and of course some practice with forking, branching, merging, and pull requests! 
+- **Wednesday 11/30** - Git'ing Ops in shape for DevOps with [Taylor Riggan](https://twitter.com/triggan)
+- **Monday 12/5** - Lessons learned from writing the DevOps Handbook with [Gene Kim](https://twitter.com/realgenekim)
+- **Wednesday 12/7** at **8P CST/9P EST** - Forking public work, committing to it, and managing the PR process with [Safia Abdalla](https://twitter.com/captainsafia)
+- **Wednesday 12/14** - Curating a group of people that build tools together with [Matthew Brender](https://twitter.com/mjbrender:)
```
Comment (rnelson0) [inline README.md]: Errant colon in your twitter link

**Comment by mbbroberg** [comment]
This is out of date due to the latest merge, which is no big deal. I'll update - thanks for the review @rnelson0. cc @jfrappier

**Review by rnelson0 — APPROVED** [review]
Assuming this is no longer a WIP I approve. Merge away! If it's still in progress, ignore this :)

**Comment by mbbroberg** [comment]
All set now - thanks for the review @rnelson0!

**Pull request merged**